data/content.json

```diff
@@ -0,0 +1,1864 @@
+{
+	"widgetPackages": [
+		{
+			"name": "Calendar2",
+			"version": "1.0.13",
+			"widgets": [
```

Comment:
An array of widgets per widget? Doesn't sound correct

Reply:
Can include more, but none of the mono repo widgets use it. Its also discouraged from pluggable widgets as the tools also dont support it.

Reply:
I chose to stick with the 'specification', even though we discourage it. A defensive move that's likely never needed, but doesn't hurt.
Since this data file is meant to be an intermediate representation anyways, we can visualize it (leave out the extra layer) in the final visualization.

Reply:
FYI @tamelbrun